yocto/ci/config-dev.yaml

```diff
@@ -2,24 +2,25 @@
 # Project configuration
 # -----------------------------
 ## Defines the name for the project
-name: mender-on-verdin
+# Set this via fly .... -v name=cpu01-bringup (or to your desired target)
+#name: cpu01-bringup
```

Comment:
Give an useful default name.

Reply:
No. it Should fail if no name is set via fly -v.

Reply:
I see. I don't like it :) 
There is a section in the readme that explains it a little bit, but this is far from: "hey, i'm new here and i know exactly what i am doing". I know that setting pipelines can be a one shot thing, but then at least provide a shell script that runs fly in a loop for all the different pipelines. If no changes in the pipelines are present, then fly would tell you so.

Reply:
Added set-dev-pipelines.sh